rally_ovs/plugins/ovs/context/ovn_nb.py

```diff
@@ -46,6 +46,7 @@ def setup(self):
 
         self.context["ovn-nb"] = lswitches
         self.context["ovs-internal-ports"] = {}
+        ovn_nbctl.close()
```

Comment:
The SSH connections to a controller/ farm node are cached(see [OvsClient.__call__()](https://github.com/ovn-org/ovn-scale-test/blob/3fbe332f96fe5881a05e0579e19e8c59f0d10b5a/rally_ovs/plugins/ovs/ovsclients.py#L53)) for reusing, so it shouldn't be closed, and next time calling self.controller_client() will return it. The purpose is improving the performance of running commands on a node by saving time to establish ssh connection. 

Once running a case is over, ssh connections will be closed before the process exit.